use_case_guide/real_time_dashboards.rst

```diff
@@ -362,6 +330,8 @@ aggregate function and its semantics. You do this by running the following:
 .. code-block:: sql
 
   -- this should be run on the workers and coordinator
+  -- (not necessary on Citus Cloud)
```

Comment:
Do we know for sure this is not necessary on Cloud? I am not sure if we automatically create the sum on hll types on Cloud formations.

Reply:
On Cloud it understood `SUM(distinct_ip_addresses)` without my having to create that aggregate.